<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainForm</class>
 <widget class="QMainWindow" name="mainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reconocimiento de iris</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Video</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="ImageWidget" name="videoWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Capturar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnForzarIdentificacion">
           <property name="text">
            <string>Forzar Identificación</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>videoWidget</zorder>
      <zorder>pushButton_4</zorder>
      <zorder>btnForzarIdentificacion</zorder>
      <zorder>label_4</zorder>
      <zorder>irisScore</zorder>
      <zorder>groupBox_2</zorder>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Imagen Capturada</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="ImageWidget" name="capturedImage" native="true">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Identificar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Registrar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Guardar imagen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Identif. automática</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>Msg. video</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="focusLabel">
           <property name="text">
            <string>Enfoque</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="focusScore">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Calidad Iris</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="irisScore">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Mensajes</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="action_Par_metros"/>
   </widget>
   <addaction name="menu_Archivo"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </action>
  <action name="action_Par_metros">
   <property name="text">
    <string>&amp;Parámetros</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header>ImageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>mainForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
